inc/TRestGeant4PhysicsLists.h

```diff
@@ -17,18 +17,17 @@
 #ifndef RestCore_TRestGeant4PhysicsLists
 #define RestCore_TRestGeant4PhysicsLists
 
+#include <TRestMetadata.h>
```

Comment:
I assume this include belongs to our local repo...
#include "TRestMetadata.h"

Reply:
It doesn't, its a dependency from the framework. If it doesn't start with `TRestGeant4` it doesn't beling to geant4lib